### PR TITLE
Geojson Layer Labels for Some Workflow Views

### PR DESCRIPTION
**Pull request by douggallup**
Primary changes in this Pull Request:

- Added capability for labeling spatial workflow result geojson layers.
- Updated map manager to handle passing the labeling data through.

Please review the checklist before submitting the Pull Request:

- [x] Pull request has a meaning full name (not just the commit message from of your last commit)
- [ ] Code has been linted using flake8
- [x] All methods have accurate Google-Style Docstrings
- [x] 100% test coverage for new content
- [ ] Tests for each common use case (please don't write one test that covers all use cases)
- [ ] Bonus: Use TypeHints

Explain deviations from original design if applicable:

### CODE REVIEW

File: tethysext/atcore/public/map_view/atcore_map_view.js [hunk 1]
```diff
@@ -159,6 +159,9 @@ var ATCORE_MAP_VIEW = (function() {
 	        }
 	    });
 
+        // TODO: Init layer labels
+        https://openlayers.org/en/latest/examples/vector-label-decluttering.html
+
```
Comment (swainn) [inline tethysext/atcore/public/map_view/atcore_map_view.js:164]: Remove these comments

**Review by swainn — CHANGES_REQUESTED** [review]
See comments.

**Comment by swainn** [comment]
Please also address the flake8 issues: https://github.com/Aquaveo/tethysext-atcore/actions/runs/6026082044/job/16348226392?pr=92

File: tethysext/atcore/public/map_view/atcore_map_view.js (hunk 1 above)
Comment (douggallup) [reply tethysext/atcore/public/map_view/atcore_map_view.js:164]: Removed (must have had an issue with merging, etc).

**Review by swainn — APPROVED** [review]

**Pull request merged**